tests/vlan/test_vlan_ping.py

```diff
@@ -160,7 +160,13 @@ def verify_icmp_packet(dut_mac, src_port, dst_port, ptfadapter):
                                              ip_dst=str(dst_port['ipv4']), ip_ttl=63)
     for i in range(5):
         testutils.send_packet(ptfadapter, src_port['port_index_list'][0], pkt)
-        testutils.verify_packet_any_port(ptfadapter, exptd_pkt, dst_port['port_index_list'])
+        try:
+            testutils.verify_packet_any_port(ptfadapter, exptd_pkt, dst_port['port_index_list'])
```

Comment:
do flush before sending and extend the timeout can not resolve the flaky?

Reply:
Extend timeout does not work.
Will try flush before sending.

Reply:
Doing flush before sending does not work too.
One observation is that only second run(send_packet and verify_packet_any_port) can work. First loop run fails.